hmmlearn/base.py

```diff
@@ -102,6 +102,9 @@ def __init__(self, n_components=1, startprob=None, transmat=None,
         self.transmat_prior = transmat_prior
         self.algorithm = algorithm
         self.random_state = random_state
+        self.logprob_ = None
+        self.n_iterations_performed = None
+        self.logprob_delta = None
```

Comment:
Yes, storing only the last `logprob_` and the delta makes sense. 

I do have one last suggestion: would you mind using a shorter name for `n_iterations_performed`, for example `n_iter_performed_`, so that it looks similar to the parameter `n_iter`?